tox.ini

```
@@ -296,6 +301,10 @@ deps =
     elasticsearch550: elasticsearch5>=5.5.0,<5.6.0
     # elasticsearch6 package
     elasticsearch640: elasticsearch6>=6.4.0,<6.5.0
+    elasticsearch70: elasticsearch7>=7.0,<7.1
+    elasticsearch71: elasticsearch7>=7.1,<7.2
+    elasticsearch75: elasticsearch7>=7.5,<7.6
+    elasticsearch76: elasticsearch7>=7.6,<7.7
```

```suggestion
    elasticsearch700: elasticsearch7>=7.0,<7.1
    elasticsearch710: elasticsearch7>=7.1,<7.2
    elasticsearch750: elasticsearch7>=7.5,<7.6
    elasticsearch760: elasticsearch7>=7.6,<7.7
```

The non-"0" name is already taken above for `elasticsearch>=7.0,<7.1`